tests/run/named-tuples.scala

```diff
@@ -0,0 +1,83 @@
+import annotation.experimental
+import language.experimental.namedTuples
+import NamedTuple.dropNames
+
+type Person = (name: String, age: Int)
+val bob = (name = "Bob", age = 33): (name: String, age: Int)
+val person2: (name: String, age: Int) = bob
+
+type Uni = (uni: Double)
+val uni = (uni = 1.0)
+val _: Uni = uni
+
+type AddressInfo = (city: String, zip: Int)
+val addr = (city = "Lausanne", zip = 1003)
+val _: AddressInfo = addr
+
+type CombinedInfo = Tuple.Concat[Person, AddressInfo]
+val bobWithAddr = bob ++ addr
+val _: CombinedInfo = bobWithAddr
+val _: CombinedInfo = bob ++ addr
+
+@main def Test =
+  println(bob)
+  println(bob.age)
+  println(person2.name)
+  println(bobWithAddr)
+  bob match
+    case p @ (name = "Bob", age = _) => println(p.age)
+  bob match
+    case p @ (name = "Peter", age = _) => println(p.age)
+    case p @ (name = "Bob", age = 0) => println(p.age)
+    case _ => println("no match")
+
+  val x = bob.age
+  assert(x == 33)
+
+  val y: (String, Int) = bob.dropNames
+
+  def ageOf(person: Person) = person.age
+
+  assert(ageOf(bob) == 33)
+  assert(ageOf((name = "anon", age = 22)) == 22)
+  assert(ageOf(("anon", 11)) == 11)
+
+  val persons = List(
+    bob,
+    (name = "Bill", age = 40),
+    (name = "Lucy", age = 45)
+  )
+  for
+    p <- persons
+    q <- persons
+    if p.age < q.age
+  do
+    println(s"${p.name} is younger than ${q.name}")
+
+  //persons.select(_.age, _.name)
+  //persons.join(addresses).withCommon(_.name)
+
+  def minMax(elems: Int*): (min: Int, max: Int) =
+    var min = elems(0)
+    var max = elems(0)
+    for elem <- elems do
+      if elem < min then min = elem
+      if elem > max then max = elem
+    (min = min, max = max)
+
+  val mm = minMax(1, 3, 400, -3, 10)
```

Comment:
will it still be possible to destructure like this? (and not need to call `.value`?)
```scala
  val (min, max) = minMax(1, 3, 400, -3, 10)
```

or we have to do this?
```scala
  val (min = min, max = max) = minMax(1, 3, 400, -3, 10)
```

Reply:
It should be supported, but is not yet implemented.